SPECS/kata-containers/kata-containers.spec

```diff
@@ -21,21 +21,16 @@
 %global kataclhdir              /usr/share/cloud-hypervisor
 %global katainitrddir           /var/cache/kata-containers/osbuilder-images/kernel-uvm
 
-%global runtime_make_vars       QEMUPATH=%{qemupath} \\\
-                                KERNELTYPE="compressed" \\\
+# DEFAULT_HYPERVISOR: makes configuration.toml link to configuration-clh.toml.
```

Comment:
Do we need this comment?

Reply:
See https://github.com/microsoft/CBL-Mariner/pull/6942#discussion_r1466721409

Reply:
I did like it for clarity. If people find it not useful, please feel free to remove